css/main.css

```diff
@@ -1,3 +1,264 @@
 /* Add your custom CSS here */
 
 /* Dont' forget to link this file to your HTML in the <head> */
+.name {
+    font-weight: bolder;
+    text-align: center;
+    margin: 8px 0px;
+}
+
+body {
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+.navlist {
+    flex-direction: column;
+    align-items: flex-start;
+}
+
+@media (min-width: 768px) {
+    .navlist {
+        display: flex;
+        flex-direction: row;
+        margin-right: -15px;
+        margin-left: -15px;
+    }
+    li {
+        text-align: center;
+    }
+}
+
+.job {
+    text-align: center;
+    margin-top: 0;
+}
+
+header {
+    background-color: antiquewhite;
+    align-items: center;
+}
+
+li {
+    margin-left: 15px;
+    list-style-type: none;
+    text-align: left;
+    font-size: 0.8rem;
+}
+
+.fa-user-circle-o {
+    color: darkcyan;
+    padding-top: 16px;
+    text-align: right;
+}
+
+.login {
+    color: rgb(39, 18, 231);
+    text-align: left;
+    font-weight: bold;
+    font-size: 0.8rem;
+}
+
+.intro {
+    background-image: url("https://unenvironment.s3.us-east-2.amazonaws.com/s3fs-public/inline-images/1.jpg");
+    background-position: center;
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-right: -15px;
+    margin-left: -15px;
+}
+
+@media (min-width: 768px) {
+    .intro {
+        min-height:500px ;
+    }
+    .ambition {
```

Comment:
This can be removed as it's the same as the `.ambition` outside of the media query

Reply:
You are totally right. I was trying to change the font size for mobile and the desktop. After I copied the whole .ambition block, forgat to correct it. So I didn't remove it, instead, corrected it. It has now different font sizes on mobile and desktop.